javascript-methods/main.js

```diff
@@ -0,0 +1,60 @@
+const num1 = 1;
+const num2 = 2;
+const num3 = 3;
+
+const maximumValue = num1 + num2 + num3;
+console.log('Maximum Value:', Math.max(maximumValue));
```

Comment:
You should be doing `Math.max` on line 5, and passing in `num1`, `num2`, & `num3`